Collision/src/main/java/dk/gruppeseks/bodtrd/collision/CollisionProcessor.java

```diff
@@ -50,6 +51,10 @@ public void process(World world)
                 {
                     continue;
                 }
+                if (ent.getState() == EntityState.DESTROYED)
```

Comment:
this should happen at the start of each of the loops. and also at the outer loop but with a continue statement